docs/clouds/openstack.md

```diff
@@ -0,0 +1,21 @@
+# Openstack
```

Comment:
Added some documentation here, but not sure how I'm supposed to generate it. I tried the makefile in the docs directory, but the output looks very not right.

Reply:
https://github.com/canonical/pycloudlib/pull/125